ovos_config/config.py

```diff
@@ -564,3 +363,16 @@ def clear_cache(message=None):
         """DEPRECATED - there is no cache anymore """
         Configuration.updated(message)
 
+
+def read_mycroft_config():
+    return dict(Configuration())
```

Comment:
Consider a docstring indicating this is a dict copy of `Configuration`

Reply:
hum, this should not be a dict, was a copy paste, i think you did this cast originally?

since Configuration() is also a dict i dont think we need this? did it break compat for you somewhere?